### DISCUSSION TITLE
Scneario Requirements missing even though our party has them (Spoilers for Scenario 3 and 8)

### DISCUSSION THREAD

**Discussion by speakanton** [Lurkars/gloomhavensecretariat#569] [Q&A] [answered]
Hi. Not sure where to ask best (BGG or here).
First of all: I love Gloomhaven Secretariat - we use it for every scenario.

My question: Why does GHS show me that I do not have fulfilled scenario requirements even though our party has the corresponding achievements.
Example: We have played scenario 3 (in GHS) and got Party Achievement "Jekserah's Plans". Afterwards, when in GHS I "set scenario" to #8 Gloomhaven Warehouse and get a popup saying that I have unfullfilled requirement "Jekserah's Plans".
Does anyone know why that is? 

settings (not sure if these play into this problem):
We play with the browser version of GHS - via ghs.champonthis.de.
In settings we have NOT active: Battle Goals, Party Sheet, Character Sheet, Character Items.

here is a screenshot of the popup
![2024-05-25_13h18_35](https://github.com/Lurkars/gloomhavensecretariat/assets/192207/af75a711-8e5a-4c9f-86f9-e124b7a417be)

**Comment by Lurkars** [comment, marked as answer]
Issue is because of disabled party sheet. I'll fix that to work as expected in future. In the meantime as workaround, just enable the party sheet and manually add the missing achievements.

**Reply by speakanton** [reply]
Thanks for the info. It worked with the setting. Also added a global achievement on the campaign part of the party sheet, which was needed for a scenario.

**Reply by Lurkars** [reply]
In [v0.97.0](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.97.0) the achievements should now be added regardless of party sheet enabled or not.
